installer/deb/scripts/preinst

```diff
@@ -23,7 +23,7 @@ display_terms_and_condition_agreement() {
     TERMS_SELECTION=$RET
 
     if [ "$TERMS_SELECTION" = "true" ]; then
-    mkdir -p $MASSASTATION_DIR
+        mkdir -m 777 -p $MASSASTATION_DIR
```

Comment:
read, write and execute access to everyone is maybe too much. is it possible to do it differently without having to create a specific user for massa station ?
My main concerns are for the serialized wallets (deletion or modification) and for WoC.

Reply:
As discussed during daily, we will open issues to discuss those subjects and merge this PR as it keeps the same behavior as it was before.